ansible/inventory/production/group_vars/all/vars.yml

```diff
@@ -6,3 +6,16 @@ website_host: privacylx.org
 
 # Lets Encrypt email
 le_email: admin@privacylx.org
+
+# Login, SSH public key, comment and state of users with admins rights
+# State absent removes the user account
+adm_acct:
+  andz:
+    login: andz
+    comment: anadahz
+    state: present
+    keys:
+      - "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIOVYO2Xu1pKrax98FpZ3dauRDMqE8Z4IPSYihD5XGRaO"
```

Comment:
can we use the vault construction to protect these? I would like my ssh public keys to remain private as they can be used as an identifier if I ever forget and reuse tem for some other project